packages/components/src/Autocomplete/Menu.tsx

```diff
@@ -50,7 +48,7 @@ export function Menu({
     <div
       className={optionMenuClass}
       ref={element => {
-        setRef(element, menuDiv);
+        menuDiv.current = element as HTMLDivElement;
```

Comment:
Are you not able to just do?

```
ref={menuDiv}
```

Reply:
It doesn't look like I can. When I try, it gives me an error since I'm using a `useRef`, which initializes with `undefined`. If I use `createRef` instead, this works fine here, but since we're using keydown listeners the state of the ref is always `undefined`, and so the desired behaviour is lost 🤔

Reply:
Just to clarify, it technically "works" using `ref={menuDiv}`, but the typescript linter blows up about it 😅 

![Screen Shot 2020-09-15 at 1 43 01 PM](https://user-images.githubusercontent.com/34727471/93262632-67921b80-f759-11ea-80ea-5a127ece3092.png)

Reply:
I _think_ if you refine your ref as follows it should clear this up.

```ts
const menuDiv = useRef() as MutableRefObject<HTMLDivElement>;
```

I suspect what is happening is typescript is unhappy that you aren't handing an undefined case that in practical terms won't come up.

Reply:
Oh yeah, that worked!! 🤩